build/gulp.docs.js

```diff
@@ -41,14 +41,20 @@ const alterReference = async () => {
 
   fs.writeFileSync(path.join(__dirname, '../docs/reference/public/modules/_botpress_sdk_.html'), newFile)
 
-  $('a').map(function() {
+  $('a').map(function () {
     const href = $(this).attr('href')
     if (href && href.startsWith('_botpress_sdk')) {
       $(this).attr('href', 'modules/' + href)
     }
+
+
+    const hrefsToReplace = ['../enums', '../classes', '../interfaces']
+    if (href && hrefsToReplace.find(x => href.startsWith(x))) {
+      $(this).attr('href', href.replace('../', '/reference/'))
```

Comment:
Use `ref.replace('../', '')` instead, otherwise the links will work on the website (absolute url) but will not work locally.  Same thing for the global file

FYI, you could declare hrefsToReplace outside the loop to avoid initializing it for each link. Also, you could merge part of that logic with the previous one (which already null-checks href)

Reply:
Nice catches, ty.
Fixed.